framework/physics/TimeStepControllers/TimeStepper.cc

```diff
@@ -0,0 +1,132 @@
+#include "TimeStepper.h"
+
+#include <cmath>
+
+namespace chi_physics
+{
+
+chi::InputParameters TimeStepper::GetInputParameters()
+{
+  chi::InputParameters params = ChiObject::GetInputParameters();
+
+  params.AddOptionalParameter("dt", 0.01, "Initial timestep to use");
+  params.AddOptionalParameter("time", 0.0, "Initial time");
+  params.AddOptionalParameter(
+    "time_index", 0, "Time index. Useful for output control.");
+  params.AddOptionalParameter("start_time", 0.0, "Start time");
```

Comment:
This `start_time` is defaulted to 0.0, the other to 1.0. Is there a reason for this?